packages/suite-native/package.json

```diff
@@ -34,6 +34,6 @@
         "@types/react": "^16.8.13",
         "@types/react-native": "^0.57.45",
         "metro-react-native-babel-preset": "^0.54.1",
-        "typescript": "^3.4.3"
+        "typescript": "^3.7.2"
```

Comment:
maybe we don't need this since we have ts defined as a dependency in root's package.json?